### ISSUE TITLE
Symptomatic multiplier

### ISSUE THREAD

**Issue by spoonerf** [Urban-Analytics/RAMP-UA#113]
@nickmalleson Would it be possible to add the multiplier for symptomatic people staying inside more to the parameters files?

Line 1487 in microsim_model.py. Also just noticed it is 0.5 - any idea when that changed from 0.1?

**Comment by spoonerf** [comment]
Ah looks like it has been like that for a while - https://github.com/Urban-Analytics/RAMP-UA/commit/fe66f696262891ddfdea93080b199c4e7ec30992#diff-440f4b8e7d175967729c88badcb9ec14

**Comment by nickmalleson** [comment]
Yes I can add a parameter, but probably not before I go on holiday. Do you mind playing around with the number in the code for now? 

https://github.com/Urban-Analytics/RAMP-UA/blob/1a4041e3142c4f78d241c18389dc77c4ddb3cb4c/microsim/microsim_model.py#L1525

Sorry no idea why it was changed from 0.5 to 0.1.

If it's going to make a big difference for you being able to use the model I don't mind chaning it before I go

**Comment by spoonerf** [comment]
Yep no worries, no rush on it can change it in the code.
